libs/dh/metering-point/feature-overview/src/lib/breadcrumb/dh-breadcrumb.component.html

```diff
@@ -15,7 +15,7 @@
 limitations under the License.
 -->
 <nav class="breadcrumb">
-  <a [routerLink]="meteringPointAbsolutePath">{{
+  <a [routerLink]="meteringPointAbsolutePath" class="watt-text-s">{{
```

Comment:
Question: is `watt-text-s` needed? I can see that you include the text styles in `libs/dh/metering-point/feature-overview/src/lib/breadcrumb/dh-breadcrumb.component.scss`

Reply:
It's necessary because the styles in the Sass file are overwritten by the global styles for links. There's a big overlap but the font-size is different. The global value is 16px and in the breadcrumb it should be 14px.